<ui version="4.0" >
 <class>MvdImportSummaryPage</class>
 <widget class="QWidget" name="MvdImportSummaryPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QStackedWidget" name="stack" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="noResultsPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="noResultsLabel" >
         <property name="text" >
          <string/>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <widget class="QProgressBar" name="progressBar" >
         <property name="value" >
          <number>100</number>
         </property>
         <property name="textVisible" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resultsPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="3" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QCheckBox" name="importMovie" >
           <property name="text" >
            <string>Import this movie</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="jumpLabel" >
           <property name="text" >
            <string>Jump to movie:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="jumpInput" />
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="3" >
        <widget class="QFrame" name="browserViewFrame" >
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="MvdBrowserView" native="1" name="jobPreview" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="MvdActionLabel" name="nextResult" >
         <property name="toolTip" >
          <string>Shows the details for the next movie that has been selected for import</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="indent" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="MvdActionLabel" name="previousResult" >
         <property name="toolTip" >
          <string>Shows the details for the previous movie that has been selected for import</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="indent" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>161</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MvdActionLabel</class>
   <extends>QLabel</extends>
   <header>actionlabel.h</header>
  </customwidget>
  <customwidget>
   <class>MvdBrowserView</class>
   <extends>QWidget</extends>
   <header>browserview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
